docs/source/models/supported_models.md

```diff
@@ -437,6 +437,11 @@ See [this page](#generative-models) for more information on how to use generativ
   * `adept/persimmon-8b-base`, `adept/persimmon-8b-chat`, etc.
   *
   * ✅︎
+- * `Plamo2ForCausalLM`
+  * PLaMo2
+  * `pfnet/plamo-2-1b`, `pfnet/plamo-2-8b`, etc.
```

Comment:
In https://huggingface.co/pfnet/plamo-2-1b/blob/main/config.json the architecture is `PlamoForCausalLM` instead of `Plamo2ForCausalLM`. Is this a mistake?

Reply:
> In https://huggingface.co/pfnet/plamo-2-1b/blob/main/config.json the architecture is `PlamoForCausalLM` instead of `Plamo2ForCausalLM`. Is this a mistake?

Actually, we have another `PlamoForCausalLM` a the different architecture (https://huggingface.co/pfnet/plamo-100b), and I used `Plamo2ForCausalLM` in vLLM to avoid misunderstandings. If it is necessary to use the same class name,  I can ask our pre-training team if it's possible to rename the config.

Reply:
> In https://huggingface.co/pfnet/plamo-2-1b/blob/main/config.json the architecture is `PlamoForCausalLM` instead of `Plamo2ForCausalLM`. Is this a mistake?

Actually, we have another `PlamoForCausalLM` a the different architecture (https://huggingface.co/pfnet/plamo-100b), and I used `Plamo2ForCausalLM` in vLLM to avoid misunderstandings. If it is necessary to use the same class name,  I can ask our pre-training team if it's possible to rename the config. I apologize for any confusion."

Reply:
IMO this is a fairly minor thing but users will see a warning message like the following, which would be nice to fix:

```You are using a model of type plamo2 to instantiate a model of type plamo. This is not supported for all configurations of models and can yield errors.```